test/module/irohad/consensus/yac/yac_proposal_storage_test.cpp

```diff
@@ -16,24 +16,19 @@ static logger::Logger log_ = logger::testLog("YacProposalStorage");
 
 class YacProposalStorageTest : public ::testing::Test {
  public:
-  YacHash hash;
-  PeersNumberType number_of_peers;
+  const PeersNumberType number_of_peers = 7;
+  const PeersNumberType supermajority = number_of_peers - number_of_peers / 5;
```

Comment:
5 is like a magic const here, so either:
1. just write 6 and leave a comment
2. create some global constant